src/components/EditSections/EditExtendedInfo/EditExtendedInfo.js

```diff
@@ -30,6 +30,10 @@ class EditExtendedInfo extends Component {
     onToggle: PropTypes.func.isRequired,
   };
 
+  state = { isFieldRequired: false };
```

Comment:
It seems, there are a lot of different fields, but only 2 of them should be marked as required in the scope of the mentioned story. Please, use a more exact name to identify the fields in which the state is controlled.